### DISCUSSION TITLE
Multiple errors after adding a new compartment

### DISCUSSION THREAD

**Discussion by ebanvil** [Open-Systems-Pharmacology/Forum#1349] [Questions & Problems]
Hello all,
I'm trying to expand the included PBPK model in PK-Sim to include a new organ (eye) using MoBi.
I'm following the instructions described in https://ascpt.onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fpsp4.12300&file=psp412300-sup-0001-suppinfo01.pdf and in this discussion (https://github.com/Open-Systems-Pharmacology/Forum/discussions/1180).

I've already performed all the steps explained above, but when I try to c**reate a simulation I get several errors**, as shown:
![image](https://user-images.githubusercontent.com/46683252/196422454-54da8e4b-cb81-4acb-88b1-3a213dc6fc3d.png)

I attach the CSV with the errors, all help is welcome
[Errores tras generar simulación.csv](https://github.com/Open-Systems-Pharmacology/Forum/files/9810053/Errores.tras.generar.simulacion.csv).

Thank you in advance,
Enrique

**Comment by AndreDlm** [comment]
Hello Enrique,

looking at the error messages it seems that many parameters that do not exist in the `spatial structure` are nonetheless referred to e.g. in the `passive transports` building block. To solve this you could manually go through the error list and check that all referenced parameters (`entities`) exist at the specified path. 
If these are standard parameters you could also have a look at how they are implemented in other default organs. 

Also note that the tutorial you referenced above was developed for small molecule drugs only, but looking at the error messages, it seems that you are working with a large molecule.

I hope this helps!

André

**Comment by wilbertdew** [comment]
Hi Enrique,

I would check if the endogenous IgG ("LigandEndo") and its FcRn complex checkbox "is present" is unchecked in all compartments of the new organ in the MSV building block.
Also, if your MSV BB is correct, make sure that the right BB is linked to the simulation (if you have multiple) and that the MSV is linked to the right BBs. This last bit is I think difficult to check afterwards, but you can select this when you create a new MSV BB or you can check the MSV values when you generate a new simulation.
